src/trajectory.jl

```diff
@@ -16,6 +16,8 @@ struct Transition{P<:PhasePoint, NT<:NamedTuple}
     stat    ::  NT
 end
 
+stat(t::Transition) = t.stat
```

Comment:
Consider renaming `stat`  to `stats`?

Reply:
I think the benefit of `stat` over `stats` is consistency with how the terms are used within the package. `stats` is currently being used for a vector of things like what `stat()` produces. I don't much care though.

Reply:
Yes I do use the convention that `stats` is for a list of `stat`.